packages/sku/tsconfig.json

```diff
@@ -15,5 +15,11 @@
     "declaration": true
   },
   "include": ["./src/**/*"],
-  "exclude": ["node_modules", "**/@loadable"]
+  "exclude": [
+    "node_modules",
+    "**/@loadable",
+    "**/*.test.ts",
+    "**/*.test.tsx",
+    "**/*.test.js"
+  ]
```

Comment:
Again, would prefer to typecheck tests if possible. I think the typical way of doing this is to have a separate `tsconfig` for building. E.g.

```json
// tsconfig.build.json
{
 "extends": "./tsconfig.json",
 "compilerOptions": {
   "outDir": "./dist"
 },
 "exclude": ["**/*.test.ts"]
}
```

And then in the build script run `tsc --project tsconfig.build.json`.